BOOKREADME.md

```diff
@@ -0,0 +1,27 @@
+# Hyland JavaScript Academy @ UCS
+Thank you for participating in this program! Please bookmark this page 😁
+
+## Replit
+[Click here to go to Replit - this is where you can write code for your websites!](https://replit.com/)
+
+## Schedule
+Material for each lesson is linked below.
+
+- [**3/2**: Kick-Off: HTML and CSS Review](HtmlCssReview/StudentDesc.md)
+- **3/9**: Introduction to JavaScript
+- **3/16**: Variables and Data Types
+- **NO ACADEMIES 3/23**
+- **3/30**: Buttons
+- **4/6**: Conditionals
+- **4/13**: Review
+- **4/20**: Libraries
+- **4/27**: Arrays
+- **5/4**: Review and Final Project Introduction
+- **5/11**: Final Project Work
+- **5/18**: Final Project Presentations
+
+## Weekly Check-in
+[Click here to complete the weekly check-in survey.](https://forms.gle/7MzJBrf2W385tAvW6)
```

Comment:
I think we will want to create a new form for this - or we can modify the existing form - or scrap it - I'm okay either way. For now, I think it's okay to just remove it from the page.

Reply:
I made a new folder and a new weekly questionnaire in google forms (but nothing else, lol) but the link here is wrong :(

I will remove this for now and we can add these links in bulk later.